packages/now-next/index.js

```diff
@@ -401,3 +438,20 @@ exports.prepareCache = async ({ cachePath, workPath, entrypoint }) => {
     ...(await glob(path.join(cacheEntrypoint, 'yarn.lock'), cachePath)),
   };
 };
+
+exports.subscribe = async ({ entrypoint, files }) => {
+  const entryDirectory = path.dirname(entrypoint);
+  const pageFiles = includeOnlyEntryDirectory(
+    files,
+    entryDirectory === '.' ? 'pages' : path.join(entryDirectory, 'pages'),
+  );
+
+  return [
+    entryDirectory === '.' ? '_next/**' : path.join(entryDirectory, '_next/**'),
```

Comment:
```suggestion
    path.join(entryDirectory, '_next/**'),
```

Both conditions return the same thing.